### PR TITLE
fix(Detailed Delivery Receipt): use dateToString method to convert dates

### PR DESCRIPTION
**Pull request by juansoos**

### CODE REVIEW

File: lib/src/message/detailed_delivery_receipt.dart [hunk 1]
```diff
@@ -14,8 +14,8 @@ class DetailedDeliveryReceipt {
 
   final String conversationSid;
   final String channelMessageSid;
-  final String dateCreatedAsDate;
-  final String dateUpdatedAsDate;
+  final String? dateCreatedAsDate;
+  final String? dateUpdatedAsDate;
```
Comment (danielgomezrico) [inline lib/src/message/detailed_delivery_receipt.dart:18]: Is it possible that these values come as empty?

**Review by danielgomezrico — APPROVED** [review]
Good job

File: lib/src/message/detailed_delivery_receipt.dart (hunk 1 above)
Comment (juansoos) [reply lib/src/message/detailed_delivery_receipt.dart:18]: In the empty tests they never arrived, but they did arrive `null`. And that's why it was the error

**Review by cdmunoz — APPROVED** [review]
🚀  🌔

**Pull request merged**